File: pset1/cash/cash.c
# include <stdio.h>
# include <cs50.h>
# include <math.h>

int main(void)
{
    float dollar;
    //prompt user for change owed
    do
    {
        dollar = get_float("change owed: ");
    }
    while (dollar <= 0);
    //converting the user’s inputted dollars to cents
    int cents = round(dollar * 100);
    int coins = 0;

    while (cents >= 1)
    {
        if (cents >= 25)
        {
            cents = cents - 25;
            coins++;
        }
        else if (cents >= 10)
        {
            cents = cents - 10;
            coins++;
        }
        else if (cents >= 5)
        {
            cents = cents - 5;
            coins++;
        }
        else if (cents >= 1)
        {
            cents = cents - 1;
            coins++;
        }
    }
    printf("%i\n", coins);


}

File: pset6/cash.py
import importlib
from cs50 import get_float


# prompt user for change owed
while True:
    try:
        dollar = get_float("change owed: ")
        if dollar > 0:
            break
    except ValueError:
        print("Enter a valid value")
# converting the user’s inputted dollars to cents
cents = round(int(dollar * 100))
coins = 0
while cents > 0:
    if cents >= 25:
        coins += 1
        cents -= 25
    elif cents >= 10:
        coins += 1
        cents -= 10
    elif cents >= 5:
        coins += 1
        cents -= 5
    elif cents >= 1:
        coins += 1
        cents -= 1
print(coins)


def user_input():
    while True:
        try:
            n = get_float("change owed: ")
            if n > 0:
                break
        except ValueError:
            print("Enter a valid value")
    count = 0
    while cents > 0:
        if cents >= 25:
            count += 1
            cents -= 25
        elif cents >= 10:
            count += 1
            cents -= 10
        elif cents >= 5:
            count += 1
            cents -= 5
        elif cents >= 1:
            count += 1
            cents -= 1


